stories/Molecules/Form/SignUp.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./signup.scss";
+
+export const variant_options = {
+  Black: "black",
+  White: "white",
+};
+
+export const SignUp = ({ label, button, placeholder, ...args }) => {
+  return (
+    <form className={["sign_up",`sign_up--${variant_options[`${args.variant}`]}`].join(" ")} action="/" id="undp-signup-subscribe">
+      <label className="footer-subtitle">{label}</label>
+      <div className="sign_up-field">
+        <input type="email" placeholder={placeholder} className="form-email" />
+        <button type="submit" className="form-submit">
```

Comment:
@Shubham-Sharma-77 We can use the button classes as it has the same theming as of link